### PR TITLE
update readme to refer to docs

### PR DESCRIPTION
**Pull request by LaurenSpiegel**
avoids duplication

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,128 +2,14 @@
 
 Boost is a tool for Filecoin storage providers to manage data storage and retrievals on Filecoin.
 
-It replaces the `lotus-miner` `markets` service, as well as the `go-fil-markets` package with a standalone binary that runs alongside a `lotus` daemon and `lotus-miner`.
+See [Documentation](https://boost.filecoin.io/) to get started.
```
Comment (jacobheun) [inline README.md:5]: Language nit
```suggestion
See the docs at [https://boost.filecoin.io](https://boost.filecoin.io/getting-started) to get started.
```

File: README.md [hunk 2]
```diff
@@ -2,128 +2,14 @@
...
 
-Boost exposes `libp2p` interfaces for making storage and retrieval deals, a web interface for managing storage deals, and a GraphQL interface for accessing and updating real-time deal information.
+## For development:
 
-## Building
 
-### Prerequisites
+1. Install using instructions in [Documentation](https://boost.filecoin.io/).
```
Comment (jacobheun) [inline README.md:10]: I think we should deep link here so people don't need to hunt around.
```suggestion
1. Install using instructions in the building and installation section in [the docs](https://boost.filecoin.io/getting-started#building-and-installing).
```

File: README.md [hunk 3]
```diff
@@ -2,128 +2,14 @@
...
-
-Arch:
-```
-sudo pacman -Syu opencl-icd-loader gcc git bzr jq pkg-config opencl-icd-loader opencl-headers opencl-nvidia hwloc
-```
-
-macOS:
-```
-brew install go bzr jq pkg-config rustup hwloc coreutils
-```
```
Comment (jacobheun) [inline README.md:30]: I don't think we're currently covering these on the docs site. If folks have already installed Lotus they shouldn't hit these problems, but I think we'll need to update on the docs site before deleting these sections.

Comment (LaurenSpiegel) [reply README.md:30]: @LexLuthr , will you please update the docs to include this or let me know how I can? thanks

Comment (LexLuthr) [reply README.md:30]: Done

Comment (LaurenSpiegel) [reply README.md:30]: thanks!  This should be ready to merge.

**Review by jacobheun — APPROVED** [review]
Looks good, thanks for updating the docs site!

**Pull request merged**